src/hotspot/share/gc/shared/collectedHeap.inline.hpp

```diff
@@ -46,4 +46,9 @@ inline oop CollectedHeap::class_allocate(Klass* klass, size_t size, TRAPS) {
   return allocator.allocate();
 }
 
+inline size_t CollectedHeap::filler_array_min_size() {
+  size_t aligned_header_size_words = heap_word_size(arrayOopDesc::base_offset_in_bytes(T_INT));
```

Comment:
Why this is called `aligned_*`? Looks like it becomes aligned only after the `align_object_size`?